packages/ui/package.json

```diff
@@ -136,6 +136,7 @@
     "sonner": "^1.7.2",
     "ts-essentials": "10.0.3",
     "use-context-selector": "2.0.0",
+    "use-effect-event": "1.0.2",
```

Comment:
We shouldn't install another package, just use the source code and give reference to the author.